examples/onboard-azure-ms-sql-server/README.md

```diff
@@ -6,8 +6,10 @@ This example creates both 'azurerm' and 'dsfhub' resources. More information reg
 - [dsfhub](https://registry.terraform.io/providers/imperva/dsfhub/latest/docs)
 
 ## Prerequisites
+Both of the following prerequisites are handled within the ``onboard-azure-eventhub`` module.
+
 ### Azure Event Hub Namespace and Event Hub
-SQL Server audit logs are sent to an Azure Event Hub and are retrieved by DSF. The Event Hubs are created inside of an Event Hub Namespace, which can contain one or more Event Hubs. Audit logs of multiple SQL Server instances can be sent to a single Event Hub. This module expects an Event Hub to have been created in advance, and will create a corresponding AZURE EVENTHUB asset for it.
+SQL Server audit logs are sent to an Azure Event Hub and are retrieved by DSF. The Event Hubs are created inside of an Event Hub Namespace, which can contain one or more Event Hubs. Audit logs of multiple SQL Server instances can be sent to a single Event Hub. This module expects an Event Hub to have been created in advance, and with a corresponding AZURE EVENTHUB asset onboarded for it.
```

Comment:
A small nitpick (and not your fault because I recognize this from our not-recently-updated docs) but could we use `within` instead of `inside of`?

Reply:
Of course, sounds better.